tests/lib/rook/base.py

```diff
@@ -70,16 +70,23 @@ def execute_in_ceph_toolbox(self, command, log_stdout=False):
     def install(self):
         # TODO(jhesketh): We may want to provide ways for tests to override
         #                 these
-        self.kubernetes.kubectl_apply(
-            os.path.join(self.ceph_dir, 'common.yaml'))
-        self.kubernetes.kubectl_apply(
-            os.path.join(self.ceph_dir, 'operator.yaml'))
+        self.kubernetes.kubectl("create namespace rook-ceph")
+        logger.info('Deploying rook operator with helm')
+        self.kubernetes.helm(f"install -n rook-ceph rook-ceph "
+                             f" {self.workspace.working_dir}/rook-ceph"
+                             f" -f {self.workspace.working_dir}/"
+                             f"rook-ceph/values.yaml")
 
         # set operator log level
         self.kubernetes.kubectl(
             "--namespace rook-ceph set env "
             "deployment/rook-ceph-operator ROOK_LOG_LEVEL=DEBUG")
 
+        self.kubernetes.kubectl_apply(
```

Comment:
Why were these moved? Shouldn't we still wait for the operator pod(s) to be running? (from line 97 below)

Reply:
Actually I didn't mean to move them but to remove them completely initially, when I realized helm chart is doing the operator onyl for now.
I 'll put same back where they were, but overall we shouldn't have to wait as this is all descriptive, k8s should take care of it.